programming/sed-article.md

```diff
@@ -747,27 +731,26 @@ The following table summarizes the most commonly used
 | ------------------------- | --------------------------------------- |
 | a _text_                  | appends _text_ after normal output      |
 | c _text_                  | changes normal output to _text_         |
-| d                         | deletes PatSpace (clears it)            |
-| D                         | deletes only the first line in PatSpace |
+| d                         | deletes the pattern space (clears it)   |
+| D                         | deletes only the first line in the pattern space |
 | i _text_                  | inserts _text_ before normal output     |
 | s/_regex_/_subex_/_flags_ | substitutes text with different text    |
 
 Recall that _subex_ is short for "substitution expression".
 
-The `d` command causes an input line to not be output,
-deleting it.
+The `d` command deletes the contents of the pattern space which prevents it from being printed.
```

Comment:
If this is the case wouldn't it print a blank line? I know it doesn't do that.
I didn't think clearing the contents of the pattern space would by itself stop it from being AutoPrinted.

Reply:
I believe it is cleared including the implicit newline.  Thus, nothing should be printed.